invenio_rdm_records/resources/serializers/iiif/schema.py

```diff
@@ -132,10 +132,18 @@ class ListIIIFFilesAttribute(fields.List):
 
     def get_value(self, obj, *args, **kwargs):
         """Return the value for a given key from an object attribute."""
+        iiif_config = current_app.config.get("IIIF_TILES_CONVERTER_PARAMS")
+        valid_metadata = (
```

Comment:
nit: I'd prefer this as a separate function for readability. But just personal preference, nothing wrong with it per se.